src/command/client/search/interactive.rs

```diff
@@ -335,6 +386,21 @@ pub fn history(
         } else {
             terminal.draw(|f| app.draw(f, &results))?;
         }
+
+        let initial_input = app.input.as_str().to_owned();
+        let initial_filter_mode = app.filter_mode;
+
+        if event::poll(Duration::from_millis(250))? {
+            while event::poll(Duration::ZERO)? {
```

Comment:
poll(0) will ensure there's an event without blocking. This mimics the `try_next()` thing we did before 👌